eoscostarica/eosio-dashboard
# feat(webapp): allow disabled menu items using env variable

### allow disabled menu items using env variable

### What does this PR do?

- Resolve #464

### Steps to test

1. change `REACT_APP_DISABLED_MENU_ITEMS` value 
2. run `make run laccahin`
3. check the sidebar menu

#### CheckList
